attestation-agent/attestation-agent/src/config/kbs.rs

```diff
@@ -20,6 +20,9 @@ pub struct KbsConfig {
 impl KbsConfig {
     pub fn new() -> Result<Self> {
         let aa_kbc_params = AaKbcParams::new()?;
+        if aa_kbc_params.kbc != "kbs" {
+            bail!("specified aa_kbc_params.kbc is not kbs");
```

Comment:
Does this mean that we aren't supporting stuff like `offline_fs_kbc` anymore?

Reply:
offline_fs_kbc is still supported in CDH side to get resource. AA's KBS config is only used to get attestation token from KBS.

Reply:
it should be `cc_kbc`, updated.